content/pt-br/shift-left.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Shift Left
+status: Completed
+category: conceito
+tags: ["metodologia", "", ""]
+---
+
+## O que é
+
+Esquerda em Shift Left se refere aos estágios iniciais no ciclo de desenvolvimento de software,
+pensando no ciclo como uma linha onde os estágios são executados da esquerda para a direita.
+Shift left é a pratica de implementar testes, segurança e outras praticas de desenvolvimento
+no início do ciclo de desenvolvimento de software e não no final.
+
+Embora originalmente usado para se referir ao processo de teste de forma antecipada,
+Shift Left agora também pode ser aplicado para outros aspectos do desenvolvimento de software e [DevOps](/devops/), como segurança e implementação (deployment).
```

Comment:
Add pt-br [DevOps](/pt-br/devops/).

Think about "como segurança e implantação". Sounds better to me. If you like it, look below.